Library/Tracking/KSRAnalytics.swift

```diff
@@ -1341,8 +1340,14 @@ public final class KSRAnalytics {
     )
   }
 
-  public func trackSignupSubmitButtonClicked() {
-    self.track(event: ApprovedEvent.signupSubmitButtonClicked.rawValue, page: .signup)
+  public func trackSignupSubmitButtonClicked(subscription: Bool) {
```

Comment:
Can you change the param name to follow the Swift convention of `Bool` beginning with `is`? So something like `isSubscribed`.